vibe/src/main/resources/templates/users/home-loggedin.html

```diff
@@ -19,13 +19,24 @@
                     <span class="barlink-text">Home</span>
                 </a>
             </li>
+            
             <li class="nav-item">

```

Comment:
hey Allen, since the form submission works, could you include that within the 'Profile' button and remove the SEND button (since that was just a placeholder)?

Reply:
done the send button gets removed. Now the profile button works